resources/views/components/footer/footer-navigation.blade.php

```diff
@@ -0,0 +1,46 @@
+<nav class="flex flex-col-reverse text-lg xl:flex-row xl:text-base">
+    <x-footer.footer-navigation-column :title="$navigation['title']">
+        <ul>
+            @foreach ($navigation['navItems'] as $item)
+                <li>
+                    <x-link-arrow-hover :url="$item['url']">
+                        {{ $item['title'] }}
+                    </x-link-arrow-hover>
+                </li>
+            @endforeach
+        </ul>
+    </x-footer.footer-navigation-column>
+    <x-footer.footer-navigation-column :title="$openingTimes['title']">
+        <table>
+            @foreach ($openingTimes['days'] as $day)
+            <tr>
+                <td class="w-32 xl:w-24">{{ $day['title'] }}</td>
+                <td>{!! $day['times'] !!}</td>
+            <tr>
+            @endforeach
+        </table>
+        <span class="block mt-5">{{ $openingTimes['text'] }} </span>
+    </x-footer.footer-navigation-column>
+    <x-footer.footer-navigation-column title="Kontakt">
+        Námeste SNP 18, 811 06 Bratislava
+        <x-link-arrow url="mailto:komarch@komarch.sk" class="mt-6 md:mt-4">
+            komarch@komarch.sk
+        </x-link-arrow>
+        <x-link-arrow-hover url="tel:+421254432080" class="mt-6 md:mt-4">
```

Comment:
according the design it looks more like 
```suggestion
        <x-link-arrow-hover url="tel:+421254432080" class="mt-0 md:mt-4">
```

from my perspective 
<img width="399" alt="Screenshot 2021-04-14 at 15 02 41" src="https://user-images.githubusercontent.com/2682941/114714511-7ada8900-9d32-11eb-9570-d395d04f76df.png">
but this is just a detail

Reply:
It depends. The desktop design keeps the spacing between them like here:

![Screenshot 2021-04-14 at 15 26 48](https://user-images.githubusercontent.com/22252756/114718070-f7229b80-9d35-11eb-8139-8569d91cbbb7.png)

So I considered this a design bug / inconsistency and falled back to the spaced version (spacing can only help on mobile interactive elements - reduces "misclicks").